src/richText.js

```diff
@@ -132,6 +134,14 @@ define([
      *        arguments are editor, ckwidget
      */
     var editor = function(input, form, options) {
+        var TRAILING_SPACE = " ";
+        if (isWebkit) {
+            // HACK use ZWS to fix cursor movement/hiding near bubble
+            TRAILING_SPACE = "\u200b ";
```

Comment:
I tried using the [7-ZWS constant](https://github.com/ckeditor/ckeditor-dev/blob/7fda187a7961d88cc162a2333ecbf821f2edc7d2/core/selection.js#L1147) (`CKEDITOR.dom.selection.FILLING_CHAR_SEQUENCE`) used by CKEditor, but it didn't work as expected. It just put 7 ZWS's in there and I could tell that they were there because I could move the cursor through or delete them (which feels like the cursor got stalled). Because of that I settled on one, which provides a better editing experience than 7.